config/shared/locales.json

```diff
@@ -1,5 +1,5 @@
 [
-  { "label": "česky", "code": "cs", "english": "Czech", "production": false },
+  { "label": "Čeština", "code": "cs", "english": "Czech", "production": false },
```

Comment:
Both work but Wikipedia uses Čeština so I went with it. https://en.wikipedia.org/wiki/Czech_language